jOOL-java-8/src/test/java/org/jooq/lambda/CollectorTests.java

```diff
@@ -990,4 +992,74 @@ public void testDropping() {
         assertEquals(Seq.of().toList(), Seq.of(1, 2, 3, 4, 5).collect(Agg.dropping(6)).toList());
         assertEquals(Seq.of("c", "d").toList(), Seq.of("a", "b", "c", "d").collect(Agg.dropping(2)).toList());
     }
+
+    //CS304 (manually written) Issue link: https://github.com/jOOQ/jOOL/issues/360
+
+    /**
+     * Test the Seq with numbers.
+     *
+     * @result The result will be the standard deviation and variance.
+     */
+    @Test
+    public void testStddevAndVarianceWithNumber() {
+        DecimalFormat df = new java.text.DecimalFormat("#.000");
+
+        Function<Integer, Double> mapping = Double::valueOf;
+
+        assertEquals(Optional.empty(), Seq.<Integer>of().collect(Agg.varianceBy(mapping)));
+        assertEquals(Optional.empty(), Seq.<Integer>of().collect(Agg.stddevBy(mapping)));
+        assertEquals(Optional.of(0.0), Seq.of(1).collect(Agg.varianceBy(mapping)));
+        assertEquals(Optional.of(0.0), Seq.of(1).collect(Agg.stddevBy(mapping)));
+        assertEquals(Optional.of(0.0), Seq.of(1, 1, 1, 1).collect(Agg.varianceBy(mapping)));
+        assertEquals(Optional.of(0.0), Seq.of(1, 1, 1, 1).collect(Agg.stddevBy(mapping)));
+        assertEquals(Optional.of(1.0), Seq.of(1, 1, 3, 3).collect(Agg.varianceBy(mapping)));
+        assertEquals(Optional.of(1.0), Seq.of(1, 1, 3, 3).collect(Agg.stddevBy(mapping)));
+        assertEquals(Optional.of(1.250), Seq.of(1, 2, 3, 4).collect(Agg.varianceBy(mapping)));
+        assertEquals(Optional.of(1.118), Optional.of(Double.parseDouble(df.format(Seq.of(1, 2, 3, 4).collect(Agg.stddevBy(mapping)).get()))));
+
+
+    }
+
+    //CS304 (manually written) Issue link: https://github.com/jOOQ/jOOL/issues/360
+
+    /**
+     * Test the Seq with numbers.
+     *
+     * @result The result will be the standard deviation and variance.
+     */
+    @Test
+    public void testStddevAndVarianceWithObject() {
+        DecimalFormat df = new java.text.DecimalFormat("#.000");
+
+        class Node {
+            final int value;
+
+            Node(int value) {
+                this.value = value;
+            }
+
+            public Double function() {
+                return (double) value;
+            }
+
+            public int getValue() {
+                return this.value;
+            }
+        }
```

Comment:
I don't understand the purpose of this `Node` class. Surely, there's an easier way to test the functionality without it.